psp/exp_configs/uk_pv.py

```diff
@@ -246,6 +265,12 @@ def get_model(self, *, random_state: np.random.RandomState | None = None) -> PvS
             random_state=random_state,
             normalize_features=True,
             capacity_getter=_get_capacity,
+            tilt_getter=_default_get_tilt
```

Comment:
Why not pass a different `tilt_getter` instead? Why add an option to the data source itself?

Reply:
yea keep it simple to just `_get_tilt`